setup.py

```diff
@@ -50,10 +50,26 @@
         ('share/snapcraft/libraries',
             ['libraries/' + x for x in os.listdir('libraries')]),
     ],
+    dependency_links=[
+        'git+git://anonscm.debian.org/apt/python-apt.git#egg=python_apt-0.0.0',
+        'https://launchpad.net/python-distutils-extra/trunk/2.39/+download/python-distutils-extra-2.39.tar.gz',
+    ],
     install_requires=[
+        'configparser',
+        'docopt',
+        'file-magic',
+        'jsonschema',
+        'oauthlib',
+        'pkg-resources',
+        'py3-progressbar',
+        'pyxdg',
+        'PyYAML',
         'pyxdg',
-        'requests',
+        'Requests',
```

Comment:
Which release of Ubuntu are you running into this on?

On Xenial I get:
error: The 'requests' distribution was not found and is required by snapcraft

I get that the first time I run './setup.py install' but the next call succeeds.